ystafdb/bin/ystafdb.py

```diff
@@ -3,22 +3,43 @@
 """YSTAFDB CLI
 
 Usage:
-  ystafdb-cli regenerate <dirpath>
+  ystafdb-cli -i <input/dirpath> -o <output/dirpath> -f <format>
 
 Options:
   -h --help     Show this screen.
   --version     Show version.
 
 """
+import argparse
 from docopt import docopt
```

Comment:
if you won't use docopt anymore for the CLI, remove this import.
docopt is fun: it allows to describe the cli options at the same time as the docstrings of the function. 
Do you have a reason why you prefer argparse ?